src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -415,20 +470,21 @@ namespace AirflowNetwork {
         Real64 ExhaustFanMassFlowRate = 0.0; // Exhaust fan flow rate used in PressureStat
         int PressureSetFlag = 0;             // PressureSet flag
         Real64 ReliefMassFlowRate = 0.0;     // OA Mixer relief node flow rate used in PressureStat
-        bool AFNDefaultControlFlag = false;  // Default simulation control flag
+        bool control_defaulted = false;  // Default simulation control flag
 
         Array1D<AirflowNetwork::AirflowNetworkNodeSimuData> AirflowNetworkNodeSimu;
         Array1D<AirflowNetwork::AirflowNetworkLinkSimuData> AirflowNetworkLinkSimu;
 
-        AirflowNetwork::AirflowNetworkSimuProp AirflowNetworkSimu;
-        // unique object name | AirflowNetwork control | Wind pressure coefficient input control | Integer equivalent for WPCCntr
-        // field | CP Array name at WPCCntr = "INPUT" | Building type | Height Selection | Maximum number of iteration |
-        // Initialization flag | Relative airflow convergence | Absolute airflow convergence | Convergence acceleration limit |
-        // Maximum pressure change in an element [Pa] | Azimuth Angle of Long Axis of Building | Ratio of Building Width Along
-        // Short Axis to Width Along Long Axis | Number of wind directions | Minimum pressure difference | Exterior large opening
-        // error count during HVAC system operation | Exterior large opening error index during HVAC system operation | Large
-        // opening error count at Open factor > 1.0 | Large opening error error index at Open factor > 1.0 | Initialization flag
-        // type
+        SimulationControl simulation_control;
+        bool distribution_simulated = false;     // True if distribution is being simulated
+        bool multizone_always_simulated = false; // True if the multizone solution is always computed
+
+        // Moved from simulation control to outer object
+        int ExtLargeOpeningErrCount = 0; // Exterior large opening error count during HVAC system operation
+        int ExtLargeOpeningErrIndex = 0; // Exterior large opening error index during HVAC system operation
+        int OpenFactorErrCount = 0;      // Large opening error count at Open factor > 1.0
+        int OpenFactorErrIndex = 0;      // Large opening error error index at Open factor > 1.0
```

Comment:
Some items from the old simulation control object make more sense out here, these values are changing and pretty much everything in the simulation control object should not.